src/lang-promql/complete/hybrid.ts

```diff
@@ -358,9 +358,11 @@ export function analyzeCompletion(state: EditorState, node: SyntaxNode): Context
 // HybridComplete provides a full completion result with or without a remote prometheus.
 export class HybridComplete implements CompleteStrategy {
   private readonly prometheusClient: PrometheusClient | undefined;
+  private readonly maxLimitMetricsMetadata: number;
 
-  constructor(prometheusClient?: PrometheusClient) {
+  constructor(prometheusClient?: PrometheusClient, maxLimitMetricsMetadata = 10000) {
```

Comment:
`max` and `limit` seems redundant. Maybe just `maxMetricsMetadata`?

Reply:
Ah yeah sounds good. I wasn't super confortable with the name of this variable

Reply:
done